misc/tests/test-standard-configs.py

```diff
@@ -18,7 +18,8 @@
 CONFIGS = {}
 CONFIGS.update(configs.default_configs_optimal(core=True, extended=True))
 CONFIGS.update(configs.default_configs_satisficing(core=True, extended=True))
-
+if "DOWNWARD_TEST_LP" in os.environ:
+    CONFIGS.update(configs.configs_optimal_lp())
```

Comment:
Currently, non-LP standard configs are tested twice. I suggest to only run LP configs if the env variable is present.